x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/googlevertexai/completion/GoogleVertexAiChatCompletionModel.java

```diff
@@ -193,13 +226,19 @@ public int rateLimitGroupingHash() {
         var projectId = getServiceSettings().projectId();
         var location = getServiceSettings().location();
         var modelId = getServiceSettings().modelId();
+        var uri = getServiceSettings().uri();
```

Comment:
Hmm I can't seem to find any information on whether the rate limiting is specific to a provider and url. Were you able to find documentation for this? Otherwise I think we should exclude them from the hash.